source/main/index.js

```diff
@@ -81,9 +81,6 @@ app.on('ready', async () => {
   try {
     await acquireDaedalusInstanceLock();
   } catch (e) {
-    const dialogTitle = 'Daedalus is unable to start!';
-    const dialogMessage = 'Another Daedalus instance is already running.';
-    dialog.showErrorBox(dialogTitle, dialogMessage);
     app.exit(1);
```

Comment:
@daniloprates I am not entirely sure that we want this `app.exit()` call here. Or that we event need try catch at all.
Please test on Windows once the build is ready to confirm the feature works as expected in terms of:
- having the first instance window being shown and focused
- having the second Daedalus instance process killed (confirmed by looking at active processes in the TaskManager).
Thanks!